src/arena/cfr/export.rs

```diff
@@ -0,0 +1,488 @@
+    
+    // Process each node in the tree
+    for node in nodes {
+        // Node styling based on type
+        let (color, shape) = match &node.data {
+            NodeData::Root => ("lightblue", "doubleoctagon"),
+            NodeData::Chance => ("lightgreen", "ellipse"),
+            NodeData::Player(_) => ("coral", "box"),
+            NodeData::Terminal(_) => ("lightgrey", "hexagon"),
+        };
+        
+        // Node label content
+        let label = match &node.data {
+            NodeData::Root => format!("Root Node\\nIndex: {}", node.idx),
+            NodeData::Chance => format!("Chance Node\\nIndex: {}", node.idx),
+            NodeData::Player(_) => {
+                // For player nodes, we need to determine which player is acting
+                // We can infer the player from the node's position in the tree
+                // Even positions after root are player 0, odd positions are player 1
+                let player_seat = if node.idx % 2 == 1 { 0 } else { 1 };
+                format!("Player {} Node\\nIndex: {}", player_seat, node.idx)
+            },
+            NodeData::Terminal(td) => format!("Terminal Node\\nIndex: {}\\nUtility: {:.2}", node.idx, td.total_utility),
+        };
+        
+        // Write node with styling
+        writeln!(
+            file,
+            "  node_{} [label=\"{}\", shape={}, style=\"filled\", fillcolor=\"{}\"];",
+            node.idx, label, shape, color
+        )?;
+        
+        // Process edges to children using the new iter_children method
+        for (child_idx, child_node_idx) in node.iter_children() {
+            let edge_label = match &node.data {
+                NodeData::Chance => {
+                    // For chance nodes, label the edge with the card
+                    let card = Card::try_from(child_idx as u8).map_or_else(
+                        |_| format!("Card {}", child_idx),
+                        |card| format!("{}", card)
+                    );
+                    card
+                },
+                NodeData::Player(_) => {
+                    // For player nodes, label the action
+                    // Index 0 is fold according to the issue description
+                    if child_idx == 0 {
+                        "Fold".to_string()
+                    } else if child_idx == 1 {
+                        "Check/Call".to_string()
+                    } else {
+                        format!("Bet/Raise {}", child_idx - 1)
+                    }
+                },
+                _ => format!("{}", child_idx),
+            };
+            
+            // Use the count for this edge to indicate frequency
+            let count = node.get_count(child_idx);
+            let edge_style = if count > 0 {
+                format!(" [label=\"{} (count: {})\", penwidth={}]", edge_label, count, 1.0 + (count as f32).min(5.0))
+            } else {
+                format!(" [label=\"{}\"]", edge_label)
+            };
```

Comment:
When possible, it would be better to use percentages. Eg we took the Call 50% of the time and Fold 10% and All in 40%.